centreon/tests/e2e/features/Dashboards/18-web-page-widget-configuration.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Configuring web page widget
+  As a Centreon User with dashboard update rights,
+  I need to configure a Web page widget on a dashboard
+  To manipulate the properties of the Web page Widget and test the outcome of each manipulation.
+
+  Scenario: Creating and configuring a Web page widget on a dashboard
+    Given a dashboard in the dashboard administrator user's dashboard library
+    When the dashboard administrator user selects the option to add a new widget
+    And the dashboard administrator user selects the widget type "Web page"
+    Then configuration properties for the Web page widget are displayed
+    When the dashboard administrator adds a valid URL
```

Comment:
I think it's necessary to test and verify the behavior about a "invalid" URL ?
Another remark, did necessary to test a HTTP & HTTPS url to confirm it's ok in rendering no ?